sopel_modules/github/webhook.py

```diff
@@ -19,6 +19,7 @@
 
 from sopel import tools
 from sopel.formatting import bold, color
+from sopel.logger import get_logger
```

Comment:
…I accidentally deleted this suggestion from 2020. Yikes.

This function is deprecated in Sopel 7.

```suggestion
```